### ISSUE TITLE
Chatlog suggestions

### ISSUE THREAD

**Issue by IstuntmanI** [Bios-Marcel/ServerBrowser#46]
- [ ] A text background colorpicker (also being able to manually introduce your favorite background's RGB code). This can make the embedded colors work better.
- [x] A checkbox if we want to display the colors as color embedding or just as plain text. (it simply works as color embedding right now)

Also, maybe adding a border to the log text ? So we would be able to have the chatlog looking more as SA-MP's chat. SA-MP's border is black by default. It looks horrible on light theme with color embedding:
![image](https://user-images.githubusercontent.com/6208948/35769386-ae1137a4-0912-11e8-9038-ed38f10c9bc1.png)
but looks almost perfect with the dark theme:
![image](https://user-images.githubusercontent.com/6208948/35769394-d2b1805a-0912-11e8-8e7d-aaf156c02507.png)

**Comment by Bios-Marcel** [comment]
I agree it looks quite shitty, i might be able to derive that is darker from the ones that are too light.

I can do the colorpicker anyways and show different colors depending on the background.

**Comment by Bios-Marcel** [comment]
Implemented CheckBox to show colorcodes as text: https://github.com/Bios-Marcel/ServerBrowser/commit/c0634ad9d2137cd6cb5c8a5b4f7981b344938888

**Comment by BigETI** [comment]
Maybe change the font to Roboto for example.

**Comment by Bios-Marcel** [comment]
Can make font configurable

**Comment by Bios-Marcel** [comment]
Instead of making customizable background colors, the color should automatically adapted.
